test/bubbleSortRecursion_test.js

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai'
+import bubbleSort from '../src/bubbleSort'
+
+describe('bubbleSort()', function() {
+
+  it('should be a function', function() {
+    expect(bubbleSort).to.be.a('function')
+  })
+
+  context("when given [9,4,6,8,7,5,1,3,2]", function() {
+    it("should return [1,2,3,4,5,6,7,8,9]", function() {
+      expect(bubbleSort([9,4,6,8,7,5,1,3,2])).to.eql([1,2,3,4,5,6,7,8,9])
```

Comment:
This should say something like: `Returns array sorted numerically: lowest to highest`